torchtext/datasets/text_classification.py

```diff
@@ -113,36 +113,88 @@ def get_vocab(self):
         return self._vocab
 
 
-def _setup_datasets(dataset_name, root='.data', ngrams=2, vocab=None, include_unk=False):
+def generate_data_iterators(dataset_name, root, ngrams, tokenizer, data_select):
```

Comment:
If this is an internal function, it is [customary](https://docs.python.org/3/tutorial/classes.html#private-variables) to use a leading underscore.

Reply:
Fixed